HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentViewModel.swift

```diff
@@ -79,18 +79,16 @@ final class PaymentComponentViewModel {
     
     // Bank image icon
     private var bankImageIconData: Data?
-    var bankImageIcon: UIImage {
-        if let bankImageIconData {
-            return UIImage(data: bankImageIconData) ?? UIImage()
-        }
-        return UIImage()
+    var bankImageIcon: UIImage? {
+        guard let bankImageIconData else { return nil }
+        return UIImage(data: bankImageIconData)
     }
 
     // Bank name label
     private var bankName: String?
-    var bankNameLabelText: String {
+    var bankNameLabelText: String? {
         if let bankName, !bankName.isEmpty {
-            return bankName
+            return nil
```

Comment:
Because in the new UI, when we do have a bank selected, we don't show anymore the name of the bank in the Payment Component View. 
I admit, is a quick fix in order not to add/pass other variables to the button configuration